src/Components/UI/modal/Modal.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import style from './Modal.module.css';
+import PropTypes from 'prop-types';
+
+const Modal = ({className, title, text, children, onModalClose}) => {
+
+  const rootClasses = [style[className]]
+
+  if (onModalClose) {
+    rootClasses.push(style.active)
+  }
+
+  return (
+    <div className={rootClasses.join(' ')} onClick={() => onModalClose()}>
+      <div className={style.modalContent} onClick={(e) => e.stopPropagation()}>
+        <div className={style.modalHead}>
+          <p>{title}</p>
+          <button onClick={() => onModalClose()}>X</button>
+        </div>
+        <p>{text}</p>
+        {children}
+      </div>
```

Comment:
NIT: I would still recommend to have cancel and submit buttons as a part of modal, but it's not necessary